src/components/MetaData/promoImageHelper.ts

```diff
@@ -6,10 +6,29 @@ export const getImgURL = (metaValue, metaType = 'og:image', globalContent, resiz
       // eslint-disable-next-line @typescript-eslint/no-var-requires,global-require
       const Thumbor = require('thumbor-lite');
       const thumbor = new Thumbor(RESIZER_SECRET_KEY, resizerURL);
-      let imgSrc = _url.replace(/^http[s]?:\/\//, '')
-        .replace(' ', '%20');
-      if (imgSrc.includes('?')) {
-        imgSrc = imgSrc.replace('?', '%3F');
+      const imgSrc = _url
+        .replace(/^http[s]?:\/\//, '')
+        .replace(' ', '%20')
+        .replace('?', '%3F');
+      /*
+        We need the focal point out of the resize options to use as a filter
+        for the thumbor image being used here
+
+        We'll just pull all the filters in case there are additional
+        restraints we want to inact
+
+        Grab the first resize option available because they all seem to have
+        the same filters
+      */
+      const resizedImageOptions = globalContent?.promo_items?.basic?.resized_params
+        || globalContent?.promo_items?.lead_art?.resized_params
+        || {};
+      const [, resizeFilters] = (Object.values(resizedImageOptions)[0] || '').match(/filters:(.*?)\//i) || [];
+      if (resizeFilters) {
+        return thumbor.setImagePath(imgSrc)
+          .resize(1200, 630)
```

Comment:
Is there a reason to hard code the image to always be resized to these width and height values? If there an image is smaller than 1200 width thumbor will upscale the image and it would result in possible blurry images being used in social sharing

Reply:
Mainly because that's what the previous code did.  I can remove it, but that would allow the image to be whatever size.

This one is over 6000x4000px:
https://corecomponents.arcpublishing.com/photo/3KFV4GL4N5DTBELYUMS7G2MP2M